client/web/src/utils/audioUtils.js

```diff
@@ -35,6 +35,7 @@ const playAudio = (
     handleFirstInteractionAudio(); // For the first interaction, we need to play a sound to unlock blend shapes
     setInitialize(false);
   }
+  handleFirstInteractionAudio();
```

Comment:
this would be done twice for if (initialize) case. Is this expected?

Reply:
Oh thanks for reminding, missing out on the code

Reply:
Pushed changes

Reply:
Thanks, please request review again once this PR is ready

Reply:
ok, everything is good